### PR TITLE
Light dark mode graph: When switching appearance mode in View Menu, Graph will change color

### PR DESCRIPTION
**Pull request by Kobsel**
### Summary

When switching appearance mode in the View Menu, the Graph will change color. "System" Appearance mode is still being developed.

---

### PR checklist

- [x] I have written tests that fail without my changes. _Or I did not add them on purpose._
- [ ] I have included instructions on how to test this, assuming they are not already apparent from the linked issue.

---

### Depends on

- [ ] Depends on #12 

### Closes/Fixes/Related to

* Related to #19

### CODE REVIEW

File: napytau/gui/gui_app.py [hunk 1]
```diff
@@ -461,12 +474,42 @@ def plot(value: int, window: GUIApp) -> Canvas:
...
     fig = Figure(figsize=(3, 2), dpi=100, facecolor="white", edgecolor="black")
 
+    #setting color
+
+    #setting color in dependence of appearance mode
+
+    if appearance == "system":
+        main_color = "#151515"
+        secondary_color = "#ffffff"
+        print("Fall1")
```
Comment (Benedikt-Brunner) [inline napytau/gui/gui_app.py:484]: These print statements should probably be removed

File: napytau/gui/gui_app.py [hunk 2]
```diff
@@ -446,12 +444,27 @@ def calc(self) -> None:
...
+
 
 def init(cli_arguments: CLIArguments) -> None:
     app = GUIApp()
     app.mainloop()
 
-def plot(value: int, window: GUIApp) -> Canvas:
+def plot(value: int, window: GUIApp, appearance: str) -> Canvas:
+
+    print(appearance)
```
Comment (Benedikt-Brunner) [inline napytau/gui/gui_app.py:467]: remove this aswell

File: napytau/gui/gui_app.py [hunk 3]
```diff
@@ -80,6 +80,8 @@ def __init__(self) -> None:
         # Initialize information area
         self._init_information_area()
 
+        self.appearance_mode.trace_add("write", self.update_plot)
```
Comment (Benedikt-Brunner) [inline napytau/gui/gui_app.py:83]: cool

**Review by Benedikt-Brunner — CHANGES_REQUESTED** [review]
Had a look at the UI, i like it 👍 
I didn't have much to say here, just some left over print statements.

These changes should be moved to the Graph Widget after it gets separated in the base PR.

Also @julius-kress needs to approve the layouting changes, as i don't have the expertise to comment on that.

**Comment by julius-kress** [comment]
> Had a look at the UI, i like it 👍 I didn't have much to say here, just some left over print statements.
> 
> These changes should be moved to the Graph Widget after it gets separated in the base PR.
> 
> Also @julius-kress needs to approve the layouting changes, as i don't have the expertise to comment on that.

Yeah I like the changes as well. Darkmode on the graph makes the GUI look much cleaner. The current graph is still a mockup, but I think we can use this when implementing the actual graph 👍

**Pull request closed without merging**